hypertrace-graphql-entity-schema/src/main/java/org/hypertrace/graphql/entity/dao/GatewayServiceEntityDao.java

```diff
@@ -24,6 +25,7 @@
   private final GraphQlGrpcContextBuilder grpcContextBuilder;
   private final GatewayServiceEntityRequestBuilder requestBuilder;
   private final GatewayServiceEntityConverter entityConverter;
+  private final BaselineProvider baselineProvider;
```

Comment:
How is BaselineProvider conceptually different than the abstract DAO interfaces we have? Do we need another concept or could this become `BaselineDao`? Provider is also already a term used in this project any any providers should implement `javax.inject.Provider`.